client/src/names.rs

```diff
@@ -0,0 +1,20 @@
+use phf::phf_map;
+
+pub static REGISTERS: phf::Map<&'static str, usize> = phf_map! {
```

Comment:
Does this naming map actually not exist anywhere else??

Reply:
If it is, I didn't manage to find it. The register names seem to be a specification-only thing, for instance `fuel-asm` and Sway inline assembly refer to them by index only.

Reply:
We have them defined here: https://github.com/FuelLabs/fuel-vm/blob/96cc3b08ff48511d88564362b99d1a1ff3d46565/src/consts.rs#L13-L60